### DISCUSSION TITLE
Restrict application/registration process to an event to logged in usergroups, but show them in calendar / list also for not logged in users

### DISCUSSION THREAD

**Discussion by woodyc79** [derhansen/sf_event_mgt#1171] [Ideas]
Currently it's possible to restrict an event to a usergroup by TYPO3's default access tab.

If a fe-group is set, only a logged in user can see that event - also in the calendar.

It would be great if there was an "apply/register" option to select a user group that is allowed to attend an event - so it would be possible for not logged in users to see those events in the calendar or in the list, but instead a "apply/register" button they would see an customizable infotext or popup that would tell the visitor that he has to register or that only registered partners are allowed to take part in the event.

Kind regards,
Christian

**Comment by derhansen** [comment]
This is not possible, since TYPO3 takes care of the access permissions based on fe_groups and it is not possible to show protected content to users without proper permission (e.g. not logged in).

**Reply by woodyc79** [reply]
Hi! Yes i know that - i did not talk about changing permissions, but adding a
* Checkbox "Only allow loggedin Users to attend this event"
* Dropdown with fe_groups
* Text for not loggedin Users
to the Backend  Event Registration Tab

So I would NOT restrict the event entry in terms of fe_groups and it would be possible for everybody to see the event itself (calendar, event list), but in terms of registration only loggedin users would be able to register for that event - the others see the Text for not loggedin users ;-)

This would be beneficials for wholesalers or companies which have education events for their partners (or future partners) and an incentive to see what education events they can participate if they where partners.

I have at least 2 customers that need that feature.

**Reply by derhansen** [reply]
But how do you then prevent non authorized users from registering to such events? If there is not fe_groups check somewhere, basically everyone can register to any event.
